corehq/apps/app_manager/views/translations.py

```diff
@@ -99,13 +103,21 @@ def download_bulk_app_translations(request, domain, app_id):
 @require_can_edit_apps
 @get_file("bulk_upload_file")
 def upload_bulk_app_translations(request, domain, app_id):
+    validate = request.POST.get('validate')
```

Comment:
i think its fine. `upload` sees what action is required and then either `process` it to update the translations or `validate` it. I think you are taking `upload` as upload translations and i am reading this as `upload` translations file.

Reply:
> You could use string_to_boolean here

Had some trouble with this. the value for `validate` is `on` if the checkbox is checked and is blank/`None` otherwise. Made the change to use `string_to_boolean` but then reverted it.

Reply:
Oh, that makes sense.  I think `string_to_boolean` is meant for interpreting user-specified strings as booleans.  If this comes from something automatic, then it'd probably make more sense to check for an exact value.

Reply:
> This parameter changes the meaning of the view

FWIW I agree with Ethan here, it'd be better to do two views (that would likely share code).